blackbox/docs/release_notes/2.0.0.txt

```diff
@@ -299,3 +302,19 @@ sets both the minimum and maximum memory to allocate, and should be set to
 whatever your previous ``CRATE_MAX_MEM`` was set to.
 
 .. _enterprise edition: https://crate.io/enterprise-edition/
+
+Cluster name in path data
+-------------------------
+
+The computation of the effective data directory path has changed in a way that
```

Comment:
This is because there's a fallback for that, which i'm describing in the last paragraph. Probably i should write
`This fallback will be removed in future versions of CrateDB` instead ?